test/new_volume_example.cpp

```diff
@@ -10,16 +10,16 @@
 #include "Eigen/Eigen"
 //#define VOLESTI_DEBUG
 #include <fstream>
-#include "random.hpp"
+#include <boost/random.hpp>
```

Comment:
do we need this file? I propose to remove and create examples in the examples directory not in tests

Reply:
for the time being of this PR you can just remove it

Reply:
I do not get it. Remove the file?

Reply:
yes remove `test/new_volume_example.cpp` but I am ok if this is not done in this PR

Reply:
let's leave it as it is for the moment